app/views/todos/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<ul>
+  <% @incomplete_todos.each do |todo| %>
+    <li>
+      <span><%= link_to todo.title, todo_path(todo, todo: {completed: true}), method: :patch %></span>
```

Comment:
Using the app, it wasn't automatically intuitive that the todo's title link was the 'complete' button. I clicked it at first thinking it was going to be the 'show' link. Not necessarily a code critic, more of a UX thing.

Reply:
We noticed afterwards that having the `complete` link on the edit page was one of the requirements, so that was definitely a miss on our part